tests/test.rfb.js

```diff
@@ -3265,6 +3265,18 @@ describe('Remote Frame Buffer protocol client', function () {
                     expect(spy).to.have.been.calledOnce;
                     expect(spy.args[0][0].detail.name).to.equal('som€ nam€');
                 });
+
+                it('should handle the extendedMouseButtons pseudo-encoding', function () {
```

Comment:
This test is redundant and a bit out of place now that the other tests properly send a pseudorect.